doc/USERMANAGEMENT.md

```diff
@@ -9,9 +9,9 @@ User Management
      * [PostgreSQL](#postgresql)
 * [3. Manage your LDAP server](#3-manage-your-ldap-server)
      * [Adding a user](#adding-a-user)
+     * [Creating groups](#Creating-groups)
 
-
-This page provides information on how to configure FADI user authentication and authorization (LDAP, RBAC, ...).
+This page provides informations on how to configure FADI user authentication and authorization (LDAP, RBAC, ...).
```

Comment:
"information" is a non-countable noun, several instances to fix in this page

https://ell.stackexchange.com/questions/17748/information-or-informations